game-app/game-headed/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -543,6 +546,9 @@ private void addTabs(HistoryDetailsPanel historyDetailPanel) {
       addTab("Actions", actionButtonsPanel, KeyCode.C);
     }
     addTab("Players", statsPanel, KeyCode.P);
+    if (!TechAdvance.getTechAdvances(data.getTechnologyFrontier(), null).isEmpty()) {
+      addTab("Technology", technologyPanel, KeyCode.E);
+    }
     addTab("Resources", economyPanel, KeyCode.R);
     if (objectivePanel != null && !objectivePanel.isEmpty()) {
```

Comment:
The private constructor ensures that when `objectivePanel.isEmpty` than `objectivePanel` is set to `null`. Then I assume we can relax this to a simple check `!= null`, right?